gstcefsrc.cc

```diff
@@ -21,14 +31,40 @@ GST_DEBUG_CATEGORY_STATIC (cef_console_debug);
 #define DEFAULT_URL "https://www.google.com"
 #define DEFAULT_GPU FALSE
 #define DEFAULT_CHROMIUM_DEBUG_PORT -1
-#define DEFAULT_LOG_SEVERITY LOGSEVERITY_DISABLE
+#define DEFAULT_LOG_SEVERITY LOGSEVERITY_DEFAULT
+#if !defined(__APPLE__) && defined(GST_CEF_USE_SANDBOX)
+#define DEFAULT_SANDBOX TRUE
+#else
 #define DEFAULT_SANDBOX FALSE
+#endif
 
+// Blocks other elements from initializing CEF is it's already in progress.
+static gboolean cef_initializing = FALSE;
```

Comment:
Turning these three variables into an enum could make this nicer:

```
typedef enum {
    CEF_STATUS_INITIAL,
    CEF_STATUS_INITIALIZING,
    CEF_STATUS_INITIALIZED,
    CEF_STATUS_SHUTTING_DOWN,
} CefStatus;
```

What do you think?

Reply:
Interesting, that looks nice!